substratest/assets.py

```diff
@@ -396,28 +404,21 @@ class Meta:
         }
 
 
-@dataclasses.dataclass
 class ComputePlan(_Asset, _ComputePlanFutureMixin):
     compute_plan_id: str
     status: str
-    traintuple_keys: typing.List[str]
-    composite_traintuple_keys: typing.List[str]
-    aggregatetuple_keys: typing.List[str]
-    testtuple_keys: typing.List[str]
+    traintuple_keys: typing.List[str] = None
+    composite_traintuple_keys: typing.List[str] = None
+    aggregatetuple_keys: typing.List[str] = None
+    testtuple_keys: typing.List[str] = None
```

Comment:
I'm not 100% but @jmorel advised to remove the `= None` as well.

Reply:
Indeed, I changed this :)